format/index.md

```diff
@@ -34,20 +34,21 @@ the header `Content-Type: application/vnd.api+json`.
 Clients **MUST** ignore all parameters for the `application/vnd.api+json`
 media type received in the `Content-Type` header of response documents.
 
+Clients that include the JSON API media type in their `Accept` header, **MUST**
+specify the media type there at least once without any media type parameters.
+
 Servers **MUST** send all JSON API data in response documents with
 the header `Content-Type: application/vnd.api+json`.
 
```

Comment:
Ok, I added this both here and in the first paragraph. I'm not 100% sure it's necessarily, but it certainly can't hurt.